vars/capsuleSystemTests.groovy

```diff
@@ -0,0 +1,196 @@
+/* groovylint-disable 
+  BuilderMethodWithSideEffects, CompileStatic, DuplicateStringLiteral, 
+  FactoryMethodName, VariableTypeRequired */
+
+
+void gitClone(String repository, String branch, String directory, String credentialsId) {
+  dir(directory) {
+    checkout([
+      $class: 'GitSCM',
+      branches: [[name: branch]],
+      userRemoteConfigs: [[
+          url: 'git@github.com:vegaprotocol/' + repository + '.git',
+          credentialsId: credentialsId
+    ]]])
+  }
+}
+
+void buildGoBinary(String directory, String outputBinary, String packages) {
+  dir(directory) {
+    sh 'go mod vendor'
+    sh 'go build -o ' + outputBinary + ' ' + packages
+  }
+}
+
+void trysh(String shellScript) {
+  try {
+    sh shellScript
+  } catch(err) {
+  }
+}
```

Comment:
I think we don't need this wrapper, you can use 
```groovy
sh script: "..." returnStatus: true
```
to get the same results: https://www.jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script